test/helper.rb

```diff
@@ -67,6 +63,10 @@ def assert_db_queries
   end
 end
 
-ActiveSupport::TestCase.fixture_path = File.dirname(__FILE__) + "/fixtures/"
+if (ActiveRecord::VERSION::MAJOR >= 7 && ActiveRecord::VERSION::MINOR >= 2) || ActiveRecord::VERSION::MAJOR >= 8
```

Comment:
It’s often easier to compare using `Gem::Version`:
```suggestion
if ActiveRecord.gem_version >= Gem::Version.new("7.2")
```